pkg/deploy/kubernetes_image_puller.go

```diff
@@ -501,7 +569,38 @@ func GetExpectedKubernetesImagePuller(ctx *DeployContext) *chev1alpha1.Kubernete
 	}
 }
 
-// Unisntall the CSV, OperatorGroup, Subscription, KubernetesImagePuller, and update the CheCluster to remove
+// UpdateDefaultImagesIfNeeded updates the default images from `spec.images` if needed
+func UpdateDefaultImagesIfNeeded(ctx *DeployContext) error {
+	specImages := StringToImageSlice(ctx.CheCluster.Spec.ImagePuller.Spec.Images)
+	csvDefaultImages := GetDefaultImages()
+	if UpdateSpecDefaults(csvDefaultImages, specImages) {
```

Comment:
Variables and function names confuse me.
`csvDefaultImages` -> `defaultImages`
`UpdateSpecDefaults` -> `UpdateSpecImages`
or could you try to find better names?

Reply:
Fixed, I think `defaultImages` and `UpdateSpecImages` are great